main-app/app/scripts/controllers/GameController.js

```diff
@@ -12,7 +12,8 @@
                     function(data){
                         $scope.gameBoard=data.gameboard;
                         if(data.outcome==='Win'){
```

Comment:
What happens at draw?
Better to handle win & draw in the "winnerDecider" service?